providers/dnsimple/dnsimpleProvider.go

```diff
@@ -360,6 +377,38 @@ func (c *DnsimpleApi) updateRecordFunc(old *dnsimpleapi.ZoneRecord, rc *models.R
 	}
 }
 
+// Returns all the zones in an account
+func (c *DnsimpleApi) ListZones() ([]string, error) {
+	client := c.getClient()
+	accountID, err := c.getAccountID()
+	if err != nil {
+		return nil, err
+	}
+
+	opts := &dnsimpleapi.ZoneListOptions{}
+	zs := []dnsimpleapi.Zone{}
```

Comment:
zs is used to collect `dnsimpleapi.Zone`'s which is used to later collect just the `Name` field from each.  Suggestion: just collect the `Name` field.

i.e. This line would become: `zones := []string` and then line 396 would become a loop that collects the zone names.  The loop around line 406 would be unneeded.

Reply:
Thanks for the feedback! Updated 🎊